bsip-0071.md

```diff
@@ -39,9 +60,12 @@ We now have several choices on handling bad debt:
 
 We need to do a deep review on bad debt before evaluating above solutions.
```

Comment:
When can we expect this deep review on bad debt? Why can't we work on improving the price feed scripts in parallel?

Reply:
Don't know. Certainly we can, care to join?